### ISSUE TITLE
Control key is incorrect when using mac.

### ISSUE THREAD

**Issue by JakeDansker** [debauchee/barrier#1141]
MacOS recognizes the windows key as the control key rather than the ctrl key on the keyboard. There is no way to change this and I would really appreciate it if this could be fixed.

**Comment by crummy** [comment]
Can you rebind it in server > configure server > double click your mac? This is what mine looks like with the rebind:

![image](https://user-images.githubusercontent.com/690626/120564163-6ecf9580-c45e-11eb-8764-9f53c22ab3ed.png)

**Comment by JakeDansker** [comment]
Yes I can. Thank you for the response. This setting is somewhat hidden and was difficult for me to find. After I changed it though it all works well. Thanks.
